src/mediasizer.js

```diff
@@ -552,10 +552,10 @@ MediaSizer = {
 
             // compatibility layer naturalWidth|Height vs. naturalHeight
             // internally width/height are used
-            options.width = options.width || options.naturalWidth;
-            options.height = options.height || options.naturalHeight;
-            options.naturalWidth = options.naturalWidth || options.width;
-            options.naturalHeight = options.naturalHeight || options.height;
+            options.width = options.width || options.naturalWidth || 100;
+            options.height = options.height || options.naturalHeight || 100;
```

Comment:
Not sure about that modification. 

It may have a bigger impact, check out the `options.hasSize` flag, which is always `true` now.

Reply:
It's only called on `init`, the impact is limited. Also, this set a default value if in any part of TAO we don't set it properly (For me, any image, video, media needs to have default values for avoid issues like no render or don't see nothing)

I will try to found other options

Reply:
Inside `mediaEditorComponent` we have defaults too.

```
var defaultConfig = {
    mediaDimension: {
        active: false
    }
}
```

Reply:
I'm not against the default value. However, could you use a constant instead of a magic value? Or better, rely on the `defaults` object: https://github.com/oat-sa/tao-core-ui-fe/pull/163/files#diff-411e90fba10497f847418435b5355f07R546

Reply:
```
var defaults = {
    disableClass: 'disabled',
    applyToMedium: true,
    denyCustomRatio: true,
    responsive: true,
    showResponsiveToggle: true,
    showReset: true,
    showSync: true,
    size: 100
};

options.width = options.width || options.naturalWidth || defaults.size;
options.height = options.height || options.naturalHeight || defaults.size;
options.naturalWidth = options.naturalWidth || options.width || defaults.size;
options.naturalHeight = options.naturalHeight || options.height || defaults.size;
```
@jsconan Like this right?

Reply:
I'm not talking about not having a default value in general, I'm talking only about this particular case.
It's obvious an undefined value of `width` and `height` was expected, at least in some cases.

Now, let's find out if they are really needed, or if we should fix the part that cannot work with the undefined value instead.

Reply:
@wazelin I'm trying to find why we pass undefined, but looking for the future, we can fix this but the bug can appear again. If we set here, a default value, doesn't matter this case or others in the future, will not appear again.

Reply:
@wazelin @andrey-ponamarev Happens here

![imagen](https://user-images.githubusercontent.com/34692207/90520367-88bb1880-e169-11ea-9eac-529404513acd.png)

`sizechange.mediasizer` is fired but interaction don't have width or height (It's on `initForm`, isn't a change like changing in properties panel)

I still suggest following my first approach for be safe of all possible bugs in future instead of fix here

Reply:
@habemuscode it is better to have a constant indeed. However, as mentioned by @wazelin, there is regression with the [hasSize property](https://github.com/oat-sa/tao-core-ui-fe/pull/163/files#diff-411e90fba10497f847418435b5355f07R561).

Reply:
Yes I plan to refactor `options.width && options.height && _.isFinite(+options.width) && _.isFinite(+options.height);` to `_.isFinite(+options.width) && _.isFinite(+options.height);`

I will not remove all, just first 2 validations.

Reply:
OK, I don't mind having the fallback values, but having them like that makes the rendering of the interaction jump for a split second.

Reply:
```
const defaultSize = 100;

        // Initialize MediaSizer
        $form.find('.media-sizer-panel').on('sizechange.mediasizer', function() {
            $(this).find('input').trigger('change');
        }).mediasizer({
            showResponsiveToggle: false,
            showSync: false,
            showReset: false,
            responsive: false,
            applyToMedium: false,
            width: interaction.object.attr('width') || defaultSize,
            height: interaction.object.attr('height') || defaultSize,
            minWidth: 50,
            maxWidth: $container.innerWidth()
        });
```

@wazelin I implement this but I don't see the difference. Can't reproduce that "jump", but OK.

I will create additional PR for that.

Reply:
https://github.com/oat-sa/extension-tao-itemqti/pull/1515